modules/peers.js

```diff
@@ -307,7 +311,6 @@ Peers.prototype.sandboxApi = function (call, args, cb) {
  * @todo rename this function to activePeer or similar
  */
 Peers.prototype.update = function (peer) {
-	peer.state = 2;
```

Comment:
ok, reverted it. Previously i was using `update` to disconnect peers as well, but I also removed it :)